schema/crdb/37.0.0/up01.sql

```diff
@@ -1,5 +1,13 @@
-CREATE TYPE IF NOT EXISTS omicron.public.live_repair_notification_type AS ENUM (
-  'started',
-  'succeeded',
-  'failed'
+-- The disposition for a particular sled. This is updated solely by the
```

Comment:
Is this showing up in my diff because of the merge (and now I see the actual 37)
where this PR now has 38?

Reply:
Yeah, I think so!